components/dashboard/src/components/ThemeSelector.tsx

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
Extracted this from `user-settings/Preferences.tsx` to re-use it.

Reply:
‼️ @selfcontained maybe something broke here? Now, visiting from the preview environment, I get a different layout in my preferences than in production (see the <kbd>Theme</kbd> section)
| gitpod.io | PE |
|----|----|
| <img width="2032" alt="image" src="https://user-images.githubusercontent.com/29888641/220938366-3aece4a7-da8c-4e25-a162-8bcccfea28da.png"> | <img width="2032" alt="image" src="https://user-images.githubusercontent.com/29888641/220938273-b8a92c9f-3ad0-491e-ac66-4befe1861e7f.png"> |

Reply:
Great catch, @filiptronicek! ⚾

Reply:
Good catch, I did adjust things a bit to make it responsive for the onboarding flow, which is funky here. I'll fix it.